deepchecks/base/display_suite.py

```diff
@@ -70,17 +91,34 @@ def display_suite_result(suite_name: str, results: List[Union[CheckResult, Check
         unique_id = ''
     else:
         unique_id = get_random_string()
+    is_widgets = is_widgets_enabled()
+    if is_widgets:
+        tab = widgets.Tab()
+        condition_tab = widgets.VBox()
+        _add_widget_classes(condition_tab)
+        checks_wo_tab = widgets.VBox()
+        _add_widget_classes(checks_wo_tab)
+        others_tab = widgets.VBox()
+        tab.children = [condition_tab, checks_wo_tab, others_tab]
+        tab.set_title(0, 'Checks With Conditions')
+        tab.set_title(1, 'Checks Without Conditions')
+        tab.set_title(2, 'Checks Without Output')
+        display_html('<style>.jupyter-widgets.widget-tab > .p-TabBar .p-TabBar-tab {flex: 0 1 auto}</style>',
+                     raw=True)
     checks_with_conditions = []
-    display_table: List[CheckResult] = []
+    checks_wo_conditions_display: List[CheckResult] = []
+    checks_w_condition_display: List[CheckResult] = []
     others_table = []
 
     for result in results:
         if isinstance(result, CheckResult):
             if result.have_conditions():
                 checks_with_conditions.append(result)
-            if result.have_display():
-                display_table.append(result)
-            else:
+                if result.have_display():
+                    checks_w_condition_display.append(result)
+            elif result.have_display():
+                checks_wo_conditions_display.append(result)
+            if not result.have_display():
```

Comment:
no need the if here, just else

Reply:
we need it because maybe it doesn't have display